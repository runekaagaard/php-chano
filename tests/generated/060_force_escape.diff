001- &lt;some html &amp; special characters &gt; here ĐÅ€£
001+ Fatal error: Call to undefined method DtlIter::force_escape() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/060_force_escape.php on line 4