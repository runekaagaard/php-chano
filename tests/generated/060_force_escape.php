<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => '<some html & special characters > here ĐÅ€£')));
foreach ($items as $i) echo $i->input->force_escape();
