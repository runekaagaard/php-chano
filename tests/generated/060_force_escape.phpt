--TEST--
A generated testfile for the "force_escape" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => '<some html & special characters > here ĐÅ€£')));
foreach ($items as $i) echo $i->input->force_escape();
--EXPECT--
&lt;some html &amp; special characters &gt; here ĐÅ€£