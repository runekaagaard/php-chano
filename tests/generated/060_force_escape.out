Fatal error: Call to undefined method DtlIter::force_escape() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/060_force_escape.php on line 4